README.md

```diff
@@ -107,6 +107,12 @@ For help on the available arguments and commands, you can run:
 teos-cli -h
 ```
 
+### Interacting remotely with teos-cli
+
+To run teos-cli remotely, you'll need to take one extra step. When teosd is started up, self-signed certificates are automatically generated for a user to make a secure connection to the Teos watchtower. When the cli is run locally, the cli knows where to find these files. But if run remotely, these files need to be copied over to the machine where the cli is being run.
```

Comment:
teos-cli -> `teos-cli`
teosd -> `teosd`
to the Teos watchtower -> to TEOS / to `teos` / to the watchtower
When the cli -> When the CLI
the cli knows -> it knows? (mainly because CLI is said two words before)
the cli is being run -> the CLI is being run